packages/blade/src/components/SkipNav/SkipNav.web.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components';
+import BaseText from '../Typography/BaseText';
+
+export const fallbackId = 'blade-skip-nav';
+// remove once VisuallyHidden is merged
+const screenReaderStyles: CSSObject = {
+  border: 0,
+  clip: 'rect(0 0 0 0)',
+  clipPath: 'inset(50%)',
+  height: '1px',
+  margin: '0 -1px -1px 0',
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  width: '1px',
+  whiteSpace: 'nowrap',
+  wordWrap: 'normal',
+};
+
+type SkipNavLinkProps = {
+  id?: string;
+  children?: string;
+};
+
+const StyledLink = styled.a(({ theme }) => ({
+  ...screenReaderStyles,
+  background: theme.colors.action.background.tertiary.default,
+  top: theme.spacing[6],
+  left: theme.spacing[6],
+  color: theme.colors.action.text.link.default,
+  ':focus': {
+    padding: theme.spacing[2],
+    clip: 'auto',
+    clipPath: 'unset',
+    width: 'auto',
+    height: 'auto',
+  },
+}));
+
+const SkipNavLink = ({
+  id = fallbackId,
+  children = 'Skip to content',
+}: SkipNavLinkProps): JSX.Element => {
+  return (
+    <StyledLink href={`#${id}`}>
+      <BaseText color="action.text.link.default" lineHeight="s">
+        {children}
+      </BaseText>
+    </StyledLink>
+  );
+};
+
+type SkipNavContentProps = {
+  id?: string;
```

Comment:
shouldn't id be required prop?

Reply:
No because generally in most cases you have one <SkipNav /> by default in a page, so requiring to add an ID for it is a suboptimal DX. Instead we fallback to `fallbackId`

Reply:
>in most cases you have one by default in a page

didn't understand this part?

Reply:
Oh I meant, in most websites you will have at least one skip navigation for sure. Thus we can set a default ID for it so that consumers don't have to worry about it. 

They can just do:  

```ts
<SkipNavLink>Skip link</SkipNavLink>
<SkipNavContent />
```